src/lib/commandBarConfigs/projectsCommandConfig.ts

```diff
@@ -26,6 +36,7 @@ export const projectsCommandBarConfig: StateMachineCommandSetConfig<
   'Open project': {
     icon: 'arrowRight',
     description: 'Open a project',
+    status: isDesktop() ? 'active' : 'inactive',
```

Comment:
There are a ton of these, we could probably throw it in a little function `activeIfIsDesktop()` (it promises they are all the same :)

Reply:
Great idea